shippable.yml

```diff
@@ -0,0 +1,53 @@
+language: none
+
+
+build:
+  pre_ci_boot:
+    image_name: ansible/ansible
+    image_tag: centos7
+    pull: true
+    # we need a terminal or certain libc functions will fail
+    options: "-ti"
+  ci:
+    # debug info
+    - id
+    - export
+# prepare the tools
+    # this image is very bare and we need cron installed to test this role
+    - yum -y install cronie crontabs
+    # the Ansible package version is far too old (1.5.4)
+    - yum -y install python-pip python-devel make gcc gcc-c++ screen
+    - pip install ansible
+    # fix a bug using the cron module
+    - yum -y install wget
+    - wget -O /tmp/ansible_cron.patch https://github.com/ansible/ansible-modules-core/commit/531e09ce167c44387fc9c6505dd0453a7251b8f9.patch
+    - patch -d $(find /usr -wholename '*/system/cron.py' | xargs dirname | xargs dirname) -p1 </tmp/ansible_cron.patch
```

Comment:
why --user? Seems easier if the whole system can count on having this version.

When I used GH tarballs, it contained only the core, so I used tarballs from http://releases.ansible.com/ansible/ instead.

Seems nice but you told me it was not in 2.1.3rc1, so I supposed it's not in 2.2.0rc2 released at the same exact time either and added the patch.

Also I would have prefered --pre to be usable. I don't see how to select a specific series (2.1.x for eg) though.